nim/happyx/config.yaml

```
@@ -1,6 +1,6 @@
 framework:
   github: HapticX/happyx
-  version: 1.7
+  version: 1.10
```

```suggestion
  version: '1.10'
```
otherwise it will show `1.1`

---

@waghanza please, see updates